app/controllers/between_categories_transactions_controller.rb

```diff
@@ -0,0 +1,43 @@
+class BetweenCategoriesTransactionsController < ApplicationController
+  before_action :authenticate_user!
+
+  def new
+    @categories = current_user.categories
+  end
+
+  def create
+    if between_categories_transaction_creator.create
+      redirect_to activity_page_path, flash: {notice: t('transaction_create')}
+    else
+      render :new
```

Comment:
@AleksSenkou If there are errors, it should be displayed in the form fields. If I add error message here, it will be displayed for each incorrect form filling, it can be annoying. 

but now I have problem with a validation error that is checked in the service.